### PR TITLE
Flux v1 end-of-life

### LINKED ISSUE #3645: Archival of Repository
I propose the repositories for Flux v1 (fluxcd/flux and fluxcd/helm-operator) are archived as of 1st November.

Reasoning behind the proposal:
- Some dependencies are pinned to EOL versions, which cannot be upgraded without causing regressions or a cascading amount of changes to the codebase. [Kustomize is one of the dependencies that cannot be upgraded](https://github.com/fluxcd/flux/issues/3500), and currently it has more than 20 CVEs, including 11 that are high or critical.
- All Kubernetes dependencies are pinned within version v1.21. That version is end-of-life support upstream.
- Users using newer versions of Kubernetes (above v1.21) are starting to [experience issues](https://github.com/fluxcd/flux/issues/3610) that we cannot resolve due to the project being in maintenance mode. The next version up ([Kubernetes v1.22](https://kubernetes.io/releases/#release-v1-22)) becomes end-of-life from 28th Oct 2022 and will no longer be supported upstream, for bug fixes nor security patches.
- The repositories not being archived send a message to the community that they are being actively maintained, which is not completely true as both repositories haven't received meaningful changes for the past 2 years, which was when they [were put on maintenance mode](https://github.com/fluxcd/flux/issues/3320).

I would like to hear from the community around concerns or issues with the proposal above, and will be closely monitoring this issue for the next week to that effect.

Relates to https://github.com/fluxcd/website/pull/1156#pullrequestreview-1121649810.

### PR DESCRIPTION
**Pull request by stefanprodan**
Update readme to reflect Flux v1 EOL, its successor and project history.

Closes: #3645

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,137 +1,44 @@
...
+and its controllers entirely.
+
+If you consider using Flux, please take a look at <https://fluxcd.io/flux/get-started>
+to get started with v2, or if you are on Flux Legacy, <https://fluxcd.io/flux/migration/> to migrate.
+
+## History
+
+Flux was initially developed by [Weaveworks](https://weave.works) and made open source in 2016.
+Over the years the community around Flux & GitOps grew and in 2019 Weaveworks decided to donate
+the project to [CNCF](https://cncnf.io).
```
Comment (dholbach) [inline README.md:15]:
```suggestion
the project to [CNCF](https://cncf.io).
```

File: README.md [hunk 2]
```diff
@@ -1,137 +1,44 @@
...
+
+In April 2020, the Flux team decided to redesign Flux from the ground up using modern
+technologies such as Kubernetes controller runtime and Custom Resource Definitions.
+The decision was made to break Flux functionally into specialised components and APIs
+with a focus on extensibility, observability and security.
+These components are now called the [GitOps Toolkit](https://fluxcd.io/flux/components/).
+
+In 2021, the Flux team lunched Flux v2 with many long-requested features like
+support for multi-tenancy, support for syncing an arbitrary number of Git repositories,
+better observability and a solid security stance. The new version made possible
```
Comment (dholbach) [inline README.md:29]:
```suggestion
better observability and a solid security stance. The new version made it possible
```

File: README.md [hunk 3]
```diff
@@ -1,137 +1,44 @@
...
+to extend Flux capabilities beyond its original GitOps design. The community rallied
+around the new design, with an overwhelming number of early adopters and
+new contributions, Flux v2 gained new features at a very rapid pace.
+
+In 2022, Flux v2 undergoes several security audits, and its code base and APIs become stable
+and production ready. Weaveworks lunched a free and open source Web UI for Flux called
```
Comment (hiddeco) [inline README.md:35]:
```suggestion
and production ready. Weaveworks launched a free and open source Web UI for Flux called
```

Comment (dholbach) [inline README.md:35]:
```suggestion
and production ready. Weaveworks launched a free and open source Web UI for Flux called
```

**Review by dholbach — APPROVED** [review]
Very nicely written, thanks @stefanprodan

**Review by pjbgf — APPROVED** [review]

**Review by hiddeco — APPROVED** [review]

**Comment by mestadler** [comment]
Good from my perspective

**Review by darkowlzz — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,137 +1,44 @@
...
+These components are now called the [GitOps Toolkit](https://fluxcd.io/flux/components/).
+
+In 2021, the Flux team launched Flux v2 with many long-requested features like
+support for multi-tenancy, support for syncing an arbitrary number of Git repositories,
+better observability and a solid security stance. The new version made it possible
+to extend Flux capabilities beyond its original GitOps design. The community rallied
+around the new design, with an overwhelming number of early adopters and
+new contributions, Flux v2 gained new features at a very rapid pace.
+
+In 2022, Flux v2 undergoes several security audits, and its code base and APIs become stable
```
Comment (kingdonb) [inline README.md:34]:
```suggestion
In 2022, Flux v2 underwent several security audits, and its code base and APIs become stable
```

**Review by kingdonb — APPROVED** [review]
LGTM, with one suggested change 👍

**Pull request merged**